apps/prairielearn/python/prairielearn/sympy_utils.py

```diff
@@ -680,7 +725,11 @@ def validate_string_as_sympy(
     custom_functions: list[str] | None = None,
     imaginary_unit: str | None = None,
 ) -> str | None:
-    """Try to parse expr as a sympy expression. If it fails, return a string with an appropriate error message for display on the frontend."""
+    """Try to parse expr as a SymPy expression. If it fails, return a string with an appropriate error message for display on the frontend.
+
+    Returns:
+        None if the expression is valid, or an error message string if it is not.
```

Comment:
Same.

Reply:
Fixed in [533ae66](https://github.com/PrairieLearn/PrairieLearn/pull/11758/commits/533ae668d695d5ca5a3f241cdcd98776a7728dbf)